geode-gfsh/src/distributedTest/java/org/apache/geode/management/internal/cli/util/MergeLogsDistributedTest.java

```diff
@@ -81,8 +81,13 @@ public void testExportInProcess() throws Exception {
     List<File> actualFiles = MergeLogs.findLogFilesToMerge(lsRule.getWorkingDirRoot());
     // remove pulse.log if present
     actualFiles =
-        actualFiles.stream().filter(x -> !x.getName().endsWith("pulse.log")).collect(toList());
-    assertThat(actualFiles).hasSize(4);
+        actualFiles.stream()
+            .filter(x -> !x.getName().endsWith("pulse.log"))
+            .filter(x -> !x.getName().contains("views"))
+            .collect(toList());
+    // besides locator views log or pulse log, there should only be locator-0.log, server-1.log and
+    // server-2.log
+    assertThat(actualFiles).hasSize(3);
```

Comment:
Consider (in both of these tests) listing the expected file names here, instead of asserting only the number of files. In addition to make sure it's the correct set of files, using the file names would make clear (without the need for a comment) why those are the expected files.